core/client/fs/src/test/java/alluxio/client/file/cache/LocalCacheManagerTest.java

```diff
@@ -612,6 +615,45 @@ public void failedPageStoreDeleteOnEviction() throws Exception {
     assertArrayEquals(PAGE1, mBuf);
   }
 
+  @Test
+  public void putTimeout() throws Exception {
+    mConf.set(PropertyKey.USER_CLIENT_CACHE_STORE_TIMEOUT, "10s");
```

Comment:
will this test take 10s? That seems too long? Would a timeout like 1s be good enough?

Reply:
changed to 2sec